Formula/llvm@11.rb

```diff
@@ -8,8 +8,8 @@ class LlvmAT11 < Formula
   revision 2
 
   livecheck do
-    url :homepage
-    regex(/LLVM (\d+\.\d+\.\d+)/i)
+    url "https://releases.llvm.org/"
+    regex(/(11(?:\.\d+)+)/i)
```

Comment:
```suggestion
    regex(/["'](11(?:\.\d+)+)["']/i)
```

It's probably best if we anchor the start/end, so we reduce the chance of matching something unexpected.